nx-X11/config/cf/X11.tmpl

```diff
@@ -251,18 +227,9 @@ XORGRELSTRING = XorgManVersionString
 #ifndef BuildScreenSaverExt
 #define BuildScreenSaverExt	NO
 #endif
-#ifndef BuildXF86VidModeExt
-#define BuildXF86VidModeExt	NO
-#endif
-#ifndef BuildXF86MiscExt
-#define BuildXF86MiscExt	NO
-#endif
```

Comment:
BuildXF86MiscExt is being removed here, but it is still referenced in `nx-X11/include/extensions/Imakefile`.

Generally, xf86misc isn't deprecated as far as I know, but still part of libXext (or split out of it.) Do you want to remove it now in order to replace libXext later with the system library?